src/thunderbots/software/ai/world/field.cpp

```diff
@@ -88,6 +88,12 @@ Rectangle Field::enemyDefenseArea() const
                      Point(field_length_ * 0.5 - defense_length_, -defense_width_ / 2.0));
 }
 
+Rectangle Field::playableArea() const
+{
+    return Rectangle(Point(-field_length_ / 2.0, -field_width_ / 2.0),
```

Comment:
We could use the existing "corner" functions for this.

Reply:
changed to 
```
    return Rectangle(friendlyCornerNeg(), enemyCornerPos());
```